src/elements/content-sidebar/stories/tests/ContentSidebar-visual.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import { type StoryObj } from '@storybook/react';
+import ContentSidebar from '../../ContentSidebar';
+import BoxAISidebar from '../../BoxAISidebar';
+
+export default {
+    title: 'Elements/ContentSidebar/tests/visual-regression-tests',
```

Comment:
we should move this under the BoxAISidebar section instead since its experimental and reliant on the BoxAI feature

Reply:
alright leave it